site/content/en/docs/manual/advanced/search-and-filters.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Search and filters'
+linkTitle: 'Search and filters'
+weight: 2
+description: 'Overview of available search options.'
+---
+
+## Search, sort and filter lists
+
+On the {{< ilink "/docs/manual/advanced/projects#projects-page" "projects" >}}, task list on the project page,
+{{< ilink "/docs/manual/basics/tasks-page" "tasks" >}}, {{< ilink "/docs/manual/basics/jobs-page" "jobs" >}},
+and {{< ilink "/docs/manual/basics/cloud-storages" "cloud storage" >}} pages, you can use search, sorting and filters.
```

Comment:
Now there more pages with filters, e.g. organization page, project page, webhook page
I do not think we should enum all pages here.